css/style.css

```diff
@@ -95,6 +95,20 @@ div.recently-dropped-toggle-container {
 }
 
 /*
+ * ban list
+ */
+
+.ban-source-toggle-container {
+  text-align: right;
+  width: 116px;
+}
+
+.ban-source-toggle-container > ul {
+  text-align: left;
+  width: 150px;
+}
```

Comment:
Ok, I was wondering about this, I 100% understand.

The div's width is the only thing keeping it from sliding all the way to the right of the page, so I guess we will have to be ok with it moving a little unless there is some CSS magic that I don't know about it. Totally possible, since I only a little CSS/HTML experience.